index.html

```diff
@@ -17,75 +17,89 @@
           </p>
-          <ul class="icons">
-            <li>
-              <a href="#"> <i class="fa-brands fa-github"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-angellist"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-facebook"></i></a>
-            </li>
-            <li>
-              <a href="#"> <i class="fa-brands fa-twitter"></i></a>
-            </li>
-          </ul>
-        </section>
-      </header>
+        </div>
+
+        <ul class="icons">
+          <li>
+            <a href="#"> <i class="fa-brands fa-github"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-linkedin-in"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-angellist"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"> <i class="fa-brands fa-twitter"></i></a>
+          </li>
+        </ul>
+      </section>
 
       <!-- Work section -->
       <section class="work-section">
-        <h1 class="work-title">My Recent Works</h1>
-        <hr />
-        <img src="img/Placeholder.png" alt="Placeholder" />
-        <h2 class="work-subtitle">Multi-Post Stories</h2>
-        <p class="work-desc">
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul class="work-section-list">
-          <li><a href="#">css</a></li>
-          <li><a href="#">html</a></li>
-          <li><a href="#">bootstrap</a></li>
-          <li><a href="#">Ruby</a></li>
-        </ul>
-        <a href="#" class="project-btn">See project</a>
+        <div class="work-title-container">
+          <h1 class="work-title">My Recent Works</h1>
+          <hr />
+        </div>
+        <div class="work-flex-container">
+          <img
+            src="img/plcholder.png"
+            alt="Placeholder"
+          />
+          <div class="work-flex-desc">
+            <h2 class="work-subtitle">Multi-Post Stories</h2>
+            <p class="work-desc">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="work-section-list">
+              <li><a href="#">css</a></li>
+              <li><a href="#">html</a></li>
+              <li><a href="#">bootstrap</a></li>
+              <li><a href="#">Ruby</a></li>
+            </ul>
```

Comment:
- [ ] please reduce the gaps between your cards
- [ ] kindly ensure all the card languages are placed the same
- [ ] please reduce the extra margin on the right of the card section. you can centralize the cards
- [ optional] you can add the see project button to all cards as it will help later to see the different projects

yours
![Screenshot (402)_LI](https://user-images.githubusercontent.com/65298048/188020405-b62f4475-9e5f-4f84-af5c-a7974459e8a2.jpg)
figma
![Screenshot (403)](https://user-images.githubusercontent.com/65298048/188020497-897317f1-8b66-4e91-be30-d847ff7f9f5d.png)

Reply:
My site columns doesnt have that much gaps between them I dont know why you are facing this issue
![screenSHot (2)](https://user-images.githubusercontent.com/96125730/188123363-895071da-dcea-4adb-90a3-4403cd5d961e.png)